funcr/funcr.go

```diff
@@ -36,14 +36,21 @@ func New(fn func(prefix, args string), opts Options) logr.Logger {
 }
 
 func newSink(fn func(prefix, args string), opts Options) logr.LogSink {
-	return &fnlogger{
+	l := &fnlogger{
 		prefix:    "",
 		values:    nil,
 		depth:     0,
 		write:     fn,
 		logCaller: opts.LogCaller,
 		verbosity: opts.Verbosity,
+		helper: opts.Helper,
```

Comment:
gofmt should have lined these up?

Reply:
Sorry for that. My gofmt integration in emacs sometimes breaks down without me noticing.